### DISCUSSION TITLE
Welcome to BAP Discussions!

### DISCUSSION THREAD

**Discussion by ivg** [BinaryAnalysisPlatform/bap#1264] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dtpeters** [comment]
Hi all, I'm a researcher out in the wilds of North Texas. I was introduced to BAP via a tool from @richinseattle who very kindly shared with me the tool and an environment to work it in.  I've since devoted quite a bit of study both to the source and some basic uses. My goals are to gain a level of competency such that I can reliably use BAP to discovery vulnerabilities in C/C++ binaries and build my own tools on top of BAP to do so.  As far as development for BAP itself goes I've only reported a couple of bugs, both of which were fixed so quickly I didn't notice until a few months later.
